server/controllers/spreadsheet-controller.js

```diff
@@ -464,6 +465,13 @@ var warningsList = {
       warningCode: "INVALID_NETWORK_SIZE",
       errorDescription: "Your network has " + genesLength + " genes, and " + edgesLength + " edges. Please note that networks are recommended to have less than 50 genes and 100 edges."
     }
+  },
+
+  incorrectlyNamedSheetWarning: function() {
+    return {
+      warningCode: "INCORRECTLY_NAMED_SHEET",
+      errorDescription: "The uploaded file appears to contain a weighted network, but contains no 'network_optimized_weights' sheet. A weighted network must be contained in a sheet called 'network_optimized_weights' in order to be drawn as a weighted graph. Please check if the sheet(s) in the uploaded spreadsheet have been named properly."
```

Comment:
Strictly speaking, we do not assume that code editors or viewers perform automatic line breaks in the same way, so the preference for long strings like this is to break them up as a concatenation, thus permitting line breaks. A natural indentation also emerges from this:

    errorDescription: "The uploaded file appears to contain a weighted network, but contains no " +
        "'network_optimized_weights' sheet. A weighted network must be contained in a sheet called " +
        "'network_optimized_weights' in order to be drawn as a weighted graph. Please check if the " +
        "sheet(s) in the uploaded spreadsheet have been named properly."